src/pages/resource/ResourcePage.tsx

```diff
@@ -27,6 +27,9 @@ const StyledResourceActionBar = styled.div`
   flex-direction: row;
   justify-content: flex-end;
   margin-top: 2rem;
+  margin-left: auto;
+  margin-right: auto;
```

Comment:
Har du sjekket i firefox? siden "auto" kalkuleres av den enkelte nettleseren.
Tipper at for alle chromium-nettlesere er det likt

Reply:
nope. will do

Reply:
det funket i FF, men gjorde det "riktig" isteden